cmd/ci/main.go

```diff
@@ -63,6 +63,12 @@ func main() {
 			{
 				Name:  "list-test-group-execution-contexts",
 				Usage: "List all test group execution contexts",
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:  "very-expensive-tests-run",
+						Usage: "Whether the groups not containing any very expensive tests should be filtered out",
```

Comment:
```suggestion
						Usage: "Whether to only include the groups with very expensive tests",
```

Reply:
Does this mean that if the boolean flag is not set that our test run includes all tests (i.e., not-very-expensive-tests + very-expensive-tests)?

Reply:
Yes, but it's worth noting that very-expensive-tests is a sub-group of not-very-expensive-tests.